### ISSUE TITLE
How to make sure it works? + Config file confusion.

### ISSUE THREAD

**Issue by DanyGee** [jcbastosportela/logioptspp#17]
Hi,

overall, the idea is gr8, but how can I make sure it works?
First, in the code page it is written I should create a `config.ini` file and then, in the binaries there is already a config file but named `conf.ini`. Which one is a valid config file?
Second, I tried it on Telegram Desktop and it doesn't smooth scroll, same in AIMP for example.
Please advise.

OS Name:                       Microsoft Windows 11 Pro
OS Version:                    10.0.26100 N/A Build 26100
Mouse:                            MX Master 3S

**Comment by DanyGee** [comment]
I've deleted both config files, launched the `logioptionspprun.exe` and the default `conf.ini` file regenerated, so probably that is the valid one. Still, instructions from the code page are confusing and should be updated and still I didn't notice smooth scroll in Telegram and AIMP. Maybe you can indicate which app would be best to test, if it actually works?

Contents of my `conf.ini` file
```
[General]
LOGI_PATH=C:\Program Files\LogiOptionsPlus\logioptionsplus_agent.exe
#BLACKLIST_APPS=explorer.exe, Taskmgr.exe, ETS6.exe, ETS6N.exe, soffice.bin, soffice.exe, dnSpy.exe
WHITELIST_APPS=Telegram.exe, AIMP.exe, notepad++.exe
#LOG_PATH=c:\tmp\log.txt
```

**Comment by MaxHasBeenUsed** [comment]
From my own experience telegram works without a problem.

Just to be sure, you want to:
1. run the official logioptionsplus.exe
2. run the logioptionspprun.exe

As for the conf.ini, the content you shared should be just fine. If you have the conf.ini as well as logioptionspp.dll and logioptionspprun.exe all under a same folder, you shouldn't have any problem getting high resolution scrolling in apps, including telegram.

**Comment by DanyGee** [comment]
I have all the needed files in `C:\Program Files\LogiOptionsPlus` folder.
I run the official `logioptionsplus.exe` -> the official Logi Options App starts.
I run the `logioptionspprun.exe` -> a window appears for a fraction of a second and disappears.
I don't see the smooth scrolling enabled 😭
I don't see anything related to `logioptionspprun.exe` running in the background processes.
Should there be anything related to `logioptionspprun.exe` running in the background?
Maybe something is blocking it from running on my machine? 🤔

@MaxHasBeenUsed 
Please check for me the following:

- anything related to `logioptionspprun.exe` running in the background for you?
- what permissions do the `logioptionspprun.exe`, `conf.ini` and `logioptionspp.dll` files have on your machine?
- in which folder do you have these files?

The logs produced the following:
```
load_configs
Applications in WHITELIST_APPS are:  
Telegram.exe 
AIMP.exe 
notepad++.exe 
DLL_PROCESS_ATTACH started
DetourAttach MyQueryFullProcessImageNameW
DLL_PROCESS_DETACH started
DetourTransactionBegin
```
In the N++ I observe some additional characters in the log:
(don't know if these have a meaning)

<img width="392" height="161" alt="Image" src="https://github.com/user-attachments/assets/48534d99-5a40-4938-a843-7283a04dcc2a" />

**Comment by MaxHasBeenUsed** [comment]
> anything related to `logioptionspprun.exe` running in the background for you?

Nothing that I can see.

> what permissions do the `logioptionspprun.exe`, `conf.ini` and `logioptionspp.dll` files have on your machine?

All typical windows default settings. Plus you don't need elevated privileges for it to work.

> in which folder do you have these files?

Some random folder outside of system drive.

Do you have smooth scroll at all, like in those 4 browsers supported by logi?

**Comment by DanyGee** [comment]
> Do you have smooth scroll at all, like in those 4 browsers supported by logi?

Yes, I do have smooth scroll in Chrome (and few other apps like for example Discord, Spotify).

What does your log produce?

**Comment by MaxHasBeenUsed** [comment]
Don't have my PC near me so can't check rn. But idk how you get smooth scrolling in Discord and Spotify. Since only those 4 browsers are supported natively, and you clearly have not whitelisted Discord and Spotify.

**Comment by DanyGee** [comment]
There must be some difference between our system environments.
We just need to pinpoint them.
Please check the logs for me once you find the time.
Thanks in advance.

**Comment by jcbastosportela** [comment]
Is this solved? Can this issue be closed?
